controllers/openstackdataplanedeployment_controller.go

```diff
@@ -87,7 +87,7 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 	defer func() { // update the Ready condition based on the sub conditions
 		if instance.Status.Conditions.AllSubConditionIsTrue() {
 			instance.Status.Conditions.MarkTrue(
-				condition.ReadyCondition, condition.ReadyMessage)
+				condition.ReadyCondition, condition.DeploymentReadyMessage)
```

Comment:
Weren't we using `ReadyMessage` to be in-line with the other operators in the OKO project?
https://github.com/openstack-k8s-operators/keystone-operator/blob/main/controllers/keystoneapi_controller.go#L147

Seems like the established standard. I can see why using "Deployment" makes sense in this context, but if all of the CR's report predictable status conditions, it might make it easier for users to follow.

Reply:
I followed the nodeset pattern: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/controllers/openstackdataplanenodeset_controller.go#L161

Reply:
The message on the NodeSet is actually NodeSet Ready

Reply:
with this PR I'm proposing Deployment Ready, but @bshephar is right, everywhere else uses the ReadyMessage. Which makes me wonder if we should change the NodeSet message